engines/wintermute/detection_tables.h

```diff
@@ -239,7 +239,12 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("actualdest", "",
 		WME_ENTRY1s("data.dcp", "6926f44b26f21ceb1d840eaab9aeb510", 9081740), Common::EN_ANY, ADGF_UNSTABLE, WME_1_8_11),
 
-	// Informer Alavi - Murder of Miss Rojan (Persian) (Try version)
+	// Informer Alavi - Murder of Miss Rojan (Persian) (Beta version -old version-, between 1387/12 to 1392/6)
+	// NOTE: This is a 2.5D game that is out of ScummVM scope
```

Comment:
Speaking of this. Maybe it makes sense to add a GF_3D or something, so we could show a message to the user? And of course, for the documentation purposes.

Reply:
@sev- For sure... I hope I'll be able to implement it in the right way